samples/cellular/nrf_provisioning/README.rst

```diff
@@ -0,0 +1,139 @@
+.. _nrf_provisioning_sample:
+
+Cellular: nRF Device provisioning
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to use the :ref:`lib_nrf_provisioning` service on your device.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9161dk_nrf9161_ns
+
+.. include:: /includes/tfm.txt
+
+The sample requires that the device's private key is installed on the device and the associated device UUID is obtained from the Identity Service.
+
+.. note::
+   This sample requires modem firmware v2.0.0 or later.
+
+Overview
+********
+
+The sample shows how the device performs the following actions:
+
+* Connects to nRF Device provisioning Service.
+* Fetches available device specific provisioning configuration.
+* Decodes the commands.
+* Acts on any AT commands, if available.
+* Reports the results back to the server.
+  In the case of an error, stops processing the commands at the first error and reports it back to server.
+* Sends ``FINISHED`` response if all the previous commands are executed without errors provided and ``FINISHED`` is one of the set provisioning commands.
+
+User interface
+**************
+
+Device side interaction is not required.
+You must define the provisioning configuration at the server side.
```

Comment:
You could link to https://docs.nrfcloud.com/SecurityServices/ProvisioningService/ProvisioningConfiguration/ProvisioningConfigurationPortal.html here.